File: sentimental-mario-less/mario.py
from cs50 import get_int


def helper(n):
    res = ""
    for i in range(1, n + 1):
        res += "#"
    return res


def helper2(n):
    res = ""
    for i in range(n):
        res += " "
    return res


height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")

for i in range(1, height + 1):
    print(helper2(height - i) + helper(i))

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    int count = 0;
    for (int i = 0; i < height; ++i)
    {
        count = 0;
        for (int j = i + 1; j < height; ++j)
        {
            printf(" ");
        }

        while (count <= i)
        {
            printf("#");
            count++;
        }

        printf("\n");
    }
}